modules/dnn/cmake/download_google_inception5.cmake

```diff
@@ -0,0 +1,30 @@
+function(download_google_inception5 dst_dir status_var)
```

Comment:
Hi, Dmitry, thank you. This makes sense. I checked on my Mac, OPENCV_TEST_DATA_PATH points at contrib_extra/testdata i.e. part of contrib_extra repository. I'm not sure if it was sent automatically or I did it manually, don't remember. This is fine but here are my questions:
1) I'd like to use space_shuttle.jpg, which is in the opencv/dnn/tutorials/images. If I copy it there it means I'm duplicating the file from opencv to opencv_extra, which is not good. On the other hand, when I'm running unit test in java, I need to find this file somehow. If it is somewhere under build/ it is easy because this is current directory for java autotests. I can also use  OPENCV_TEST_DATA_PATH but this is less attractive because this variable may be not defined - do I have to skip test in this case?
2) OPENCV_DNN_TEST_DATA_PATH is empty on my Mac. If I want my tests to be included in auto-builds, I need to where this var is pointing to on auto-build environments. 
Sorry if my questions are a bit unclear.